lib/json_string.dart

```diff
@@ -5,7 +5,9 @@
 /// A simple and lightweight JSON data container for Dart and Flutter.
 library json_string;
 
-export 'src/errors.dart';
+export 'package:json_util/src/errors.dart'
+    show JsonFormatException, JsonEncodingError, JsonDecodingError;
```

Comment:
JsonFormatException, JsonEncodingError, JsonDecodingError still visible from outside but sources moved to json_util.json_util